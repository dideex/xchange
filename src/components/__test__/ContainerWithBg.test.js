import React from 'react'
import Component from '../ContainerWithBg'
import {mount} from 'enzyme'

describe('Container with bg', () => {
  const Node = () => <div>Test children</div>
  it('Basic markup', () => {
    const wrapper = mount(
      <Component>
        <Node />
      </Component>,
    )

    expect(wrapper.html()).toMatchSnapshot()
  })
})
