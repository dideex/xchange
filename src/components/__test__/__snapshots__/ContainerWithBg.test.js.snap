// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container with bg Basic markup 1`] = `"<section class=\\"css-rz5030\\"><svg class=\\"\\" style=\\"z-index: -1; position: absolute; width: 100%; left: 0px; right: 0px;\\" viewBox=\\"0 0 1443 468\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><path style=\\"fill: #40c0cb;\\" d=\\"M1442.5 467.502C1442.5 355.002 1307 109 0.000183105 0.501465V467.502H1442.5Z\\"></path></svg><div>Test children</div><svg class=\\"\\" style=\\"z-index: -1; position: absolute; width: 100%; left: 0px; right: 0px;\\" viewBox=\\"0 0 1445 101\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><path style=\\"fill: #40c0cb;\\" d=\\"M1444.5 0.499512C926 89.9998 653.5 82.9999 0 100.5V0.499512H1444.5Z\\"></path></svg></section>"`;
